xieffect/test/communities/tasks/question_test.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+from flask_fullstack import dict_cut, SocketIOTestClient
+
+from communities.tasks.tests_db import Question, QuestionKind
+from test.conftest import FlaskTestClient, delete_by_id
+
+
+@pytest.fixture()
+def question_data(test_id: int) -> dict[str, Any]:
+    return {
+        "text": "Test question",
+        "test_id": test_id,
+        "kind": QuestionKind.CHOICE.to_string(),
+    }
+
+
+@pytest.fixture()
+def question_id(question_data: dict[str, Any]) -> int:
+    question: Question = Question.create(
+        **dict(question_data, kind=QuestionKind.CHOICE)
+    )
+    yield question.id
+    delete_by_id(question.id, Question)
+
+
+@pytest.fixture()
+def question_ids(test_community: int, test_id: int, question_id: int) -> dict[str, int]:
+    return {
+        "community_id": test_community,
+        "test_id": test_id,
+        "question_id": question_id,
+    }
+
+
+def test_create_question(
+    client: FlaskTestClient,
+    socketio_client: SocketIOTestClient,
+    test_id: int,
+    test_community: int,
+    question_data: dict[str, Any],
+):
+    data: dict[str, Any] = {
+        "community_id": test_community,
+        **question_data,
+    }
+
+    socketio_client.assert_emit_ack(
+        event_name="new_question",
+        data=data,
+        expected_data={**dict_cut(data, "kind", "text"), "id": int},
```

Comment:
Мы сначала добавили в question_data community_id, потом отрезали... Тут можно так без изменения смысла:
```suggestion
        expected_data={**dict_cut(question_data, "kind", "text"), "id": int},
```

Reply:
И тогда можно будет убрать переменную data